### PR TITLE
documentation

### PR DESCRIPTION
**Pull request by garrett-vangilder**
## Status
**READY/IN DEVELOPMENT**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

Adds documentation

## Resolves Issue Number

documentation

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Visit the CartController and the ProductSubTypeController

## Impacted Files in Application
List general components of the application that this PR will affect:

*

### CODE REVIEW

File: Controllers/CartController.cs [hunk 1]
```diff
@@ -28,12 +28,14 @@ namespace BangazonWeb.Controllers
     public class CartController : Controller
     {   
         private BangazonContext context;
-
+
```
Comment (daynewright) [inline Controllers/CartController.cs:32]: Need documentation for constructor

File: Controllers/CartController.cs [hunk 2]
```diff
@@ -28,12 +28,14 @@ namespace BangazonWeb.Controllers
...
         private BangazonContext context;
-
+        
         public CartController(BangazonContext ctx)
         {
             context = ctx;
         }
 
+
+    /* Adds logged in user and checks view model.  Sends view model to view file. */
```
Comment (daynewright) [inline Controllers/CartController.cs:39]: Does not follow documentation formatting

**Review by daynewright — DISMISSED** [review]
A few updates for documentation

**Comment by daynewright** [comment]
LGTM

File: Controllers/CartController.cs (hunk 2 above)
Comment (MattKraatz) [reply Controllers/CartController.cs:39]: 👍

**Review by MattKraatz — APPROVED** [review]
LGTM except for that one CartController thang

**Comment by Anulfo** [comment]
LGTM

**Review by Anulfo — COMMENTED** [review]
LGTM

**Pull request merged**